docs/Reference/LTS.md

```diff
@@ -23,6 +23,12 @@ in this document:
    given Fastify release line. This implies that only the latest Node.js release
    of a given line is supported.
 
+4. Major releases will be tested and verified against all [N|solid](https://doc
+   s.nodesource.com/nsolid/5.0/docs) releases lines those released aligned with
+   the [Node.js LTS policy](https://github.com/nodejs/Release) within the LTS 
+   period of that given Fastify release line. This implies that only the latest
+   N|Solid release of a given line is supported.
```

Comment:
The first sentence does not make any sense to me.

Reply:
Hi @jsumners, you're right this is a bit confusing.
I've pushed a more comprehensive text about what should be the LTS policy for those alternative runtimes.
Let me know if it makes sense now.
Thanks.